code/modules/client/client_defines.dm

```diff
@@ -1,10 +1,10 @@
 /client
-		//////////////////////
-		//BLACK MAGIC THINGS//
-		//////////////////////
-	parent_type = /datum
+	parent_type = /datum		// black magic
+	preload_rsc = PRELOAD_RSC	// This is 0 so we can set it to an URL once the player logs in and have them download the resources from a different server.
+	view = WORLD_VIEW
+	var/view_radius = WORLD_VIEW_NUM
+	var/datum/tooltip/tooltips
```

Comment:
why are tooltips cached in a list? Shouldn't they just be part of screen generally?

Reply:
> why are tooltips cached in a list? Shouldn't they just be part of screen generally?

Because this show only for clients. And all mobs have hud.